index.json

```diff
@@ -49642,6 +50937,114 @@
       "edited",
       "removed_from_repository"
     ],
+    "properties": {
+      "team": {
+        "type": "object",
+        "description": "The team itself."
+      },
+      "changes": {
+        "type": "object",
+        "description": {
+          "0": "T",
+          "1": "h",
+          "2": "e",
+          "3": " ",
+          "4": "c",
+          "5": "h",
+          "6": "a",
+          "7": "n",
+          "8": "g",
+          "9": "e",
+          "10": "s",
+          "11": " ",
+          "12": "t",
+          "13": "o",
+          "14": " ",
+          "15": "t",
+          "16": "h",
+          "17": "e",
+          "18": " ",
+          "19": "t",
+          "20": "e",
+          "21": "a",
+          "22": "m",
+          "23": " ",
+          "24": "i",
+          "25": "f",
+          "26": " ",
+          "27": "t",
+          "28": "h",
+          "29": "e",
+          "30": " ",
+          "31": "a",
+          "32": "c",
+          "33": "t",
+          "34": "i",
+          "35": "o",
+          "36": "n",
+          "37": " ",
+          "38": "w",
+          "39": "a",
+          "40": "s",
+          "41": " ",
+          "42": "`",
+          "43": "e",
+          "44": "d",
+          "45": "i",
+          "46": "t",
+          "47": "e",
+          "48": "d",
+          "49": "`",
+          "50": ".",
```

Comment:
When I apply the following diff
```diff
diff --git a/lib/section-to-webhook.ts b/lib/section-to-webhook.ts
index 2a0a264..2fd1a40 100644
--- a/lib/section-to-webhook.ts
+++ b/lib/section-to-webhook.ts
@@ -76,12 +76,13 @@ const getProperties = ($: cheerio.Root): Webhook["properties"] => {
     },
     obj: { [key: string]: any }
   ) => {
-    const matches = [...str.matchAll(/(\w+)/g)];
-    return matches.reduce((cursor, [, key], index) => {
+    const matches = Array.from(str.matchAll(/(\w+)/g), m => m.slice(1,4));
+    return matches.reduce((cursor, [key, array, arrayKey], index) => {
+      const propKey = key ?? arrayKey;
       if (index + 1 === matches.length) {
-        return (cursor[key] = value);
+        return (cursor[propKey] = value);
       }
-      return (cursor[key] = { ...cursor[key] });
+      return (cursor = { 'type': array === '[]' ? 'array' : 'object', ...cursor });
     }, obj);
   };
```
I get the expected result of having a string as the property and not what it currently is `{ ...'string' }`

Any pointers welcome